bin/install

```diff
@@ -66,10 +66,12 @@ install_lua() {
 
         # If we are installing Lua 5.x or greater install LuaRocks as well
         if version_5x_or_greater $version; then
-            echo "Installing LuaRocks..."
-            curl -L 'https://luarocks.org/releases/luarocks-3.1.3.tar.gz' --output luarocks.tar.gz || exit 1
+            local luarocks_version=$(get_latest_luarocks_version)
+            local luarocks_name=luarocks-$luarocks_version
```

Comment:
Looks like we need to check for `ASDF_LUA_LUAROCKS_VERSION` here in order to see if the user has overridden the default of the latest version.

Reply:
Weird. I have it here, but didn't commit it. Oops!